azure-pipelines.yml

```diff
@@ -1056,7 +1056,7 @@ jobs:
 
 - job: CheckHeaders
   displayName: "Check License, History, and DocMap"
-
+  condition: and(not(contains(variables['Build.SourceBranch'], 'lts')),not(contains(variables['System.PullRequest.TargetBranch'],'lts')))
```

Comment:
In LTS PR:
`Expanded: and(not(contains('refs/pull/30848/merge', 'lts')), not(contains('dev-lts-2.66', 'lts')))`

In batched CI, `System.PullRequest.TargetBranch` is an empty string, and the condition is still correct.
```
Evaluating: and(not(contains(variables['Build.SourceBranch'], 'lts')), not(contains(variables['System.PullRequest.TargetBranch'], 'lts')))
Expanded: and(not(contains('refs/heads/lts-skip-version-check', 'lts')), not(contains(variables['System.PullRequest.TargetBranch'], 'lts')))
```